packages/block-library/src/form/index.php

```diff
@@ -35,9 +35,19 @@ function render_block_core_form( $attributes, $content ) {
 
 	$extra_fields = apply_filters( 'render_block_core_form_extra_fields', '', $attributes );
 
+	// Remove styles and classes, so they can be properly added to the wrapper using `get_block_wrapper_attributes()`.
+	$processed_content->remove_attribute( 'style' );
+	$processed_content->remove_attribute( 'class' );
+
 	return str_replace(
-		'</form>',
-		$extra_fields . '</form>',
+		array(
+			'<form',
+			'</form>',
+		),
+		array(
+			'<form ' . get_block_wrapper_attributes(),
+			$extra_fields . '</form>',
+		),
```

Comment:
I'm not sure I understand whats happening here / how this is related to the change. Maybe some more descriptive comment about why we need to do this would be helpful :)

Reply:
Ah my bad... This code was part of https://github.com/WordPress/gutenberg/pull/55755 and it slipped in this one. I rebased the PR keeping only the commit relevant to this change.
Thank you for catching this! 👍